src/Gir/Marshal/SymbolTable.cs

```diff
@@ -72,20 +86,20 @@ static string TrimConstAndPointer (string type)
 			return type.Substring (start, end - start);
 		}
 
-		public ISymbol this [string type] {
-			get {
-				var actualType = TrimConstAndPointer (type);
```

Comment:
TrimConstAndPointer is not used anymore now but I kept the method for now. Do we want to remove it?

Reply:
Since this is a new project lets be proactive in removing stuff that isn't being used anymore.

Reply:
Remove it!